ts/Core/Chart/StockChart.ts

```diff
@@ -926,10 +941,124 @@ Series.prototype.init = function (): void {
     // Call base method
     seriesInit.apply(this, arguments as any);
 
-    // Set comparison mode
-    this.initCompare(this.options.compare as any);
+    if (this.options.compare) {
+        // Set comparison mode
+        this.initCompare(this.options.compare);
+    } else if (this.options.cumulative) {
+        // Set Cumulative Sum mode
+        this.initCumulative(this.options.cumulative);
+    }
+};
+
+/**
+ * Shared code for the axis.setCompare() and the axis.setCumulative() methods.
+ * @private
+ */
+Axis.prototype.setModifier = function (
+    mode: 'compare'|'cumulative',
+    modeState?: boolean|string,
```

Comment:
I think there is no handler for the case when `modeState: null` 

`typeof modeState === 'object'`